service/src/test/kotlin/fi/lempaala/evaka/invoice/service/ProEInvoiceGeneratorTest.kt

```diff
@@ -13,6 +13,10 @@ import org.junit.jupiter.api.Test
 import org.mockito.kotlin.mock
 import org.mockito.kotlin.whenever
 
+private val restrictedStreetAddress = "Turvakielto"
```

Comment:
The restricted address needs to be the default finance decision address for the municipality. The hard coded address used here is a Tampere specific address.

Reply:
Changed restricted address from "Turvakielto" to default decision address